pkg/receiver/rawk8seventsreceiver/README.md

```diff
@@ -0,0 +1,41 @@
+# Raw Kubernetes Events Receiver
+
+Receiver for ingesting Kubernetes Events in their raw format, exactly as the Kubernetes API returns them.
+It intends to return exactly the same output as the following [FluentD plugin].
+
+Supported pipeline types: logs
+
+> :construction: This receiver is in **ALPHA**. Configuration fields, behaviour and log data model are subject to change.
+
+## Configuration
+
+```yaml
+receivers:
+  raw_k8s_events:
+    # Namespaces to collect events from. Empty array means all namespaces and is the default.
+    # default = []
+    namespaces: []
+
+    # Maximum age of collected events relative to receiver start time.
+    # The default setting of 1 minute means that we're going to collect events up to 1 minute backwards in time.
+    # default = 1m
+    max_event_age: 1m
+
+    # Authentication type for the Kubernetes client. Valid values are: `serviceAccount`, `kubeConfig`, `tls` and `none`.
+    # default = `serviceAccount`
+    auth_type: serviceAccount
```

Comment:
It would be nice to point for some configuration details (how to change host, port and so on)

Reply:
You can't change any of that in this configuration though. Host and port are read from standard Kubernetes environment variables.

Reply:
I know it, but there is no indication which envs can be changed here

Reply:
I don't know if I want to reproduce Kubernetes client documentation here, which this would come down to.